### ISSUE TITLE
Battery level inconsistency

### ISSUE THREAD

**Issue by grayver** [grayver/mi-vigem#4]
My Xiaomi Gamepad has turned off due to low battery level. However it reported 61% battery level over data protocol.. Need to figure this out.

**Comment by zhangbo8418** [comment]
Is it fixed?

**Comment by grayver** [comment]
No. Currently I translate battery level value which I receive from gamepad. I will collect statistics and decide whether this value needs to be transformed.. If you have the same experience, please share.
